src/v2/view-builder/utils/LinksUtil.js

```diff
@@ -29,6 +29,11 @@ const getSwitchAuthenticatorLink = (appState) => {
   return [];
 };
 
+const shouldShowForgotPasswordLink = (appState) => {
```

Comment:
@prad] I needed some clarification on this, with this are you suggesting to move the rendering logic for Forgot Password inside BaseFooter instead?

Reply:
responded over slack

Reply:
I've updated the logic now so it's more generic. A note that I'm defining the base method inside BaseFooter as it doesn't make sense to keep it there since the related logic doesn't live there either. Instead I have it under IdentifierFooter.